SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/MappedOntologyMappingLoader.cs

```diff
@@ -46,16 +46,22 @@ public OntologyMapping LoadOntologyMapping()
                     using (StreamReader reader = new StreamReader(stream))
                     {
                         string result = reader.ReadToEnd();
-                        var mappings = JsonSerializer.Deserialize<OntologyMapping>(result, options);
-
-                        if (mappings != null)
+                        OntologyMapping? mappings;
+                        try
                         {
-                            return mappings;
+                            mappings = JsonSerializer.Deserialize<OntologyMapping>(result, options);
                         }
-                        else
+                        catch (JsonException jex)
                         {
-                            throw new MappingFileException("Mappings file is empty or poorly formed.", resourcePath);
+                            throw new MappingFileException($"Mappings file '{resourcePath}' is malformed.", resourcePath, jex);
```

Comment:
Because this is in the "partner defined" project it is separated by a Nuget from the base. Any changes to the base need to be published as a nuget and consumed by the partner implementations (*.Mapped, etc) at "their" convenience (its a burden for us because it takes two PR loops to accomplish this sort of change). 

(That is why you dont see a ProjectReference in the csproj)